q2lint.py

```diff
@@ -35,11 +35,18 @@ def main():
                           pathlib.Path('.').glob('**/*/package.json'))
     if npm_packages:
         import subprocess
+        import os
 
         for package in npm_packages:
             pkg_path = package.parent
-            cmd = 'cd %s && npm i && npm run build -- --bail && (git diff ' \
-                  '--quiet */bundle.js || bash -c "exit 42")' % pkg_path
+            install = 'cd %s && npm i --silent' % pkg_path
+            cmd = subprocess.run(install, shell=True,
+                                 stdout=open(os.devnull, 'wb'))
+            if cmd.returncode != 0:
+                errors.append('npm install failed: %s' % cmd.stderr)
```

Comment:
Just to be clear, the `continue` on [line 47](https://github.com/qiime2/q2lint/pull/8/files#diff-f9ee26d996666a083d01926e85c789d1R47) *should* (as is) stop the `build` step from being reached on anything that fails to install.  So if the `install` fails, I don't think we will get any useful information from the `build` step unless we fix the `install` step.

Is this the desired behavior?  Your comment seems to suggest that maybe the code should allow the `build` step to happen even if the `install` step fails.  Or am I misinterpreting your point (or misreading the code)?